frontend/src/components/samples/SampleDetailProcess.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import {connect} from "react-redux";
+import {Link} from "react-router-dom";
+
+import {Table, Tag} from "antd";
+
+import {withUser, withProtocol} from "../../utils/withItem";
+
+
+const mapStateToProps = state => ({
+  usersByID: state.users.itemsByID,
+  protocolsByID: state.protocols.itemsByID,
+});
+
+const SampleDetailProcess = ({processes, usersByID, protocolsByID,}) => {
+    const columns = [
+      {
+        title: '',
+        dataIndex: 'id',
+        key: 'id',
+        render: (id, _) =>
+          <Link to={`/processes/${id}`}>
+            Process #{id}
+          </Link>,
+      },
+      {
+        title: 'Execution Date',
```

Comment:
Just to confuse you some more ... I would call it "Date Processed", like in the processSampleList. Execution Date does not sound as "processy".